client/gulpfile.js

```diff
@@ -57,6 +61,26 @@ const sources = {
   }
 };
 
+gulp.task('scss:compileJsToScss', () => {
+  const variablesConfigPath = 'scss/utilities/variables_config';
+  const compiledVariablesPath = 'scss/utilities/compiled_variables';
+  const files = fs.readdirSync(variablesConfigPath);
+
+  if (!fs.existsSync(compiledVariablesPath)) {
```

Comment:
Is there a benefit? We don't need to add another module to the client code if we stick with `fs`.